pycontrails/core/models.py

```diff
@@ -72,6 +75,16 @@ class ModelParams:
     #: Experimental. See :mod:`pycontrails.core.interpolation`.
     interpolation_use_indices: bool = False
 
+    #: Experimental. Alternative interpolation method to account for specific humidity
+    #: lapse rate bias. Must be one of "linear-q", "cubic-spline", or "log-q-log-p".
+    #: The "linear-q" method applies naive linear interpolation over gridded met data.
+    #: The "cubic-spline" method applies a custom stretching of the met interpolation
+    #: table to account for the specific humidity lapse rate bias. The "log-q-log-p"
+    #: method interpolates in the log of specific humidity and pressure, then converts
+    #: back to specific humidity.
+    #: Only used by models calling to :func:`interpolate_met`.
+    interpolation_q_method: str = "linear-q"
```

Comment:
Does it complicate things if this is set by default to None? 

The default interpolation method should be whatever the overarching `interpolation_method` is -- the user could specify `bilinear` or `cubic` as the interpolation method for the whole grid as its written.

Reply:
When we decide to set a default, then this would allow the user to turn it off by setting the value to None -- that feels more intuitive.

Reply:
Thanks -- I replaced "linear-q" with None.

c190074612edb81b7716e1c83bc6034f97652be9